lib/ui/view/expandable_textfield/expandable_textfield_screen.dart

```diff
@@ -224,8 +227,8 @@ class ExpandableTextFieldScreen extends HookWidget {
                                               if (textController.text
                                                   .toString()
                                                   .isNotEmpty) {
-                                                sendMessage(
-                                                    textController.text);
+                                                sendMessage(textController.text,
+                                                    model.mediaList);
```

Comment:
Please make media optional.
What you did means I must send media with text which should not be so

Reply:
changes made